Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeNumberFormat.cpp

```diff
@@ -308,7 +308,7 @@ static ErrorOr<void> parse_number_system_digits(String core_supplemental_path, U
 
 static String parse_identifiers(String pattern, StringView replacement, UnicodeLocaleData& locale_data, NumberFormat& format)
 {
-    static Utf8View whitespace { "\u0020\u00a0\u200f"sv };
+    static Utf8View constexpr whitespace { "\u0020\u00a0\u200f"sv };
```

Comment:
Please don't start east-constexpr, it's not even allowed there by the standard (see http://eel.is/c++draft/dcl.pre#nt:simple-declaration - `specifier-seq` only allowed before declarator)

Reply:
Hmm, it seems to compile fine, but sure, I'll change it.